src/main/kotlin/org/rust/cargo/toolchain/flavors/RsMacToolchainFlavor.kt

```diff
@@ -11,7 +11,7 @@ import java.nio.file.Path
 
 object RsMacToolchainFlavor : RsToolchainFlavor() {
     override fun getHomePathCandidates(): List<Path> {
-        val path = "/usr/local/Cellar/rust/bin".toPath()
```

Comment:
Shouldn't we check the old path as well to maintain the compatibility?

Reply:
Homebrew <3.0 adds symlinks for installed binaries to `/usr/local/bin`, which is already covered by `RsUnixToolchainFlavor`.

Reply:
Homebrew 3.0 `arm` and `x86-64` builds can be installed side-to-side on an M1 machine. Will `x86-64` build also add the symlinks?